assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py

```diff
@@ -532,18 +574,24 @@ def annotate_batch(iterator):
                     client = mlflow.tracking.MlflowClient()
                     client.log_artifacts(
                         child_run_id, DEFAULT_PROMPTFLOW_PATH, artifact_path=artifact_path)
-            tabular_result = pd.read_json(os.path.join(output_dir.name, "eval_results.jsonl"), lines=True)
+            # convert rows from result to pandas dataframe
+            tabular_result = pd.DataFrame(tabular_result["rows"])
             for passthrough_column, passthrough_values in passthrough_cols.items():
                 tabular_result[passthrough_column] = passthrough_values
             # rename metric columns
             try:
                 for column_name in metrics_list:
                     # set failures to -1
-                    tabular_result[column_name] = pd.to_numeric(tabular_result[column_name], errors='coerce')
-                    tabular_result[column_name].fillna(-1, inplace=True)
-                    tabular_result.rename(
-                        columns={column_name: COLUMN_TO_COMPACT_METRIC_NAME[column_name]},
-                        inplace=True)
+                    metric_name_compact = COLUMN_TO_COMPACT_METRIC_NAME[column_name]
+                    # output column names follow schema like "outputs.coherence.gpt_coherence"
+                    result_name = ("outputs." + metric_name_compact + "." + column_name).lower()
+                    tabular_result[result_name] = pd.to_numeric(tabular_result[result_name], errors='coerce')
+                    tabular_result[result_name].fillna(-1, inplace=True)
+                    tabular_result.rename(columns={result_name: metric_name_compact}, inplace=True)
+                for column_name in input_columns:
+                    # input column names follow schema like "inputs.context"
+                    result_name = "inputs." + column_name
+                    tabular_result.rename(columns={result_name: column_name}, inplace=True)
```

Comment:
Just want to confirm: would the evaluation table looks exactly same as the old version? Because we have some dependency later to consume that table

Reply:
yes it looks exactly the same - there is no change to the output... just the library used to calculate the metrics is different, promptflow-evals instead of azure-ai-generative (which will be deprecated afaik)